src/lib/components/Search/index.js

```diff
@@ -107,10 +108,28 @@ class Search extends BaseElement {
 
   /* eslint-disable indent */
   get hitsTemplate() {
-    if (!(this.hits && this.hits.length && this.showHits)) {
+    if (!this.showHits) {
       return "";
     }
 
+    if (!this.hits.length) {
+      if (!this.query) {
+        return "";
+      }
+
+      const query = "web.dev " + this.query.trim();
```

Comment:
Did you want to scope search results just to web.dev? If so I think you could do:

```
const query = "site:web.dev " + this.query.trim();
```

Reply:
I intentionally didn't. I think Algolia is pretty good and Google finds our site most of the time anyway, but this just broadens the possible answers for users.

I've added a comment.

Reply:
And an Analytics event.